sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -45,13 +45,24 @@
 ]
 
 
-class _MetricFunctionWrapper(BaseEstimator):
-    def __init__(self, func, name=None, greater_is_better=False):
-        self._func = func
-        self.name = name if name is not None else func.__name__
-        self.greater_is_better = greater_is_better
+class _BaseForecastingErrorMetric(BaseMetric):
+    """Base class for defining forecasting error metrics in sktime. Extends
+    sktime's BaseMetric to the forecasting interface.
+    """
+
+    _tags = {
+        "requires_y_train": False,
```

Comment:
Let's only use hyphens for separating words in tags for consistency

Reply:
Will do; switching "requires_y_train" to "requires-y-train".